js/binary/reader.js

```diff
@@ -389,27 +389,25 @@ jspb.BinaryReader.prototype.skipFixed64Field = function() {
  */
 jspb.BinaryReader.prototype.skipGroup = function() {
   // Keep a stack of start-group tags that must be matched by end-group tags.
-  var nestedGroups = [this.nextField_];
+  var previousField = this.nextField_;
```

Comment:
Why is this change? Can you separate it out from this pull request? Also please add tests if you are fixing a bug.

Reply:
The benchmark will fail to parse if don't fix this bug.

So submit the bug fix first?

Reply:
Yes, please do that in a separate PR with unit-tests.